package.json

```diff
@@ -15,75 +15,85 @@
     "test:debug": "node --inspect-brk node_modules/.bin/vue-cli-service test:unit --testPathPattern --no-cache --watch --runInBand"
   },
   "dependencies": {
-    "@babel/compat-data": "^7.19.1",
+    "@babel/compat-data": "^7.21.5",
     "@bcrs-shared-components/breadcrumb": "2.1.11",
     "@bcrs-shared-components/confirm-dialog": "1.2.1",
     "@bcrs-shared-components/corp-type-module": "1.0.11",
     "@bcrs-shared-components/court-order-poa": "2.1.4",
     "@bcrs-shared-components/document-delivery": "1.2.1",
-    "@bcrs-shared-components/enums": "1.0.37",
+    "@bcrs-shared-components/enums": "1.0.38",
     "@bcrs-shared-components/folio-number-input": "1.1.17",
-    "@bcrs-shared-components/interfaces": "1.0.54",
+    "@bcrs-shared-components/interfaces": "1.0.60",
     "@bcrs-shared-components/staff-comments": "1.3.11",
     "@bcrs-shared-components/staff-payment": "2.1.11",
     "@mdi/font": "^5.9.55",
     "@sentry/integrations": "^7.50.0",
     "@sentry/vue": "^7.50.0",
     "axios": "^0.27.2",
-    "core-js": "^3.8.3",
+    "core-js": "^3.30.1",
     "country-list": "^2.2.0",
     "http-status-codes": "^2.2.0",
-    "launchdarkly-js-client-sdk": "^2.22.1",
+    "launchdarkly-js-client-sdk": "^2.24.2",
     "lodash": "^4.17.21",
-    "pinia": "^2.0.33",
+    "lodash.omit": "^4.5.0",
+    "pdfjs-dist": "3.4.120",
+    "pinia": "^2.0.35",
     "pinia-class": "^0.0.3",
-    "pdfjs-dist": "2.0.943",
     "provinces": "^1.11.0",
     "qrcode.vue": "1.7.0",
-    "regenerator-runtime": "^0.13.9",
+    "regenerator-runtime": "^0.13.11",
     "register-service-worker": "^1.7.2",
-    "sbc-common-components": "3.0.5",
-    "vue": "^2.7.10",
+    "sbc-common-components": "3.0.8",
+    "vue": "^2.7.14",
```

Comment:
Use 2.7.10 for now - Severin found some issues with 2.7.14

Reply:
Deal, on it 👍

Reply:
This is resolved.